packages/safe-core-sdk/src/utils/safeVersions.ts

```diff
@@ -1,31 +1,24 @@
 import semverSatisfies from 'semver/functions/satisfies'
 
-export enum FEATURES {
-  SAFE_TX_GAS_OPTIONAL,
-  SAFE_TX_GUARDS,
-  SAFE_FALLBACK_HANDLER,
-  ETH_SIGN
+export enum SAFE_FEATURES {
+  SAFE_TX_GAS_OPTIONAL = 'SAFE_TX_GAS_OPTIONAL',
+  SAFE_TX_GUARDS = 'SAFE_TX_GUARDS',
+  SAFE_FALLBACK_HANDLER = 'SAFE_FALLBACK_HANDLER',
+  ETH_SIGN = 'ETH_SIGN'
 }
 
-const FEATURES_BY_VERSION: Record<string, string> = {
-  [FEATURES.SAFE_TX_GAS_OPTIONAL]: '>=1.3.0',
-  [FEATURES.SAFE_TX_GUARDS]: '>=1.3.0',
-  [FEATURES.SAFE_FALLBACK_HANDLER]: '>=1.1.1',
-  [FEATURES.ETH_SIGN]: '>=1.1.0'
+const SAFE_FEATURES_BY_VERSION: Record<SAFE_FEATURES, string> = {
+  [SAFE_FEATURES.SAFE_TX_GAS_OPTIONAL]: '>=1.3.0',
+  [SAFE_FEATURES.SAFE_TX_GUARDS]: '>=1.3.0',
+  [SAFE_FEATURES.SAFE_FALLBACK_HANDLER]: '>=1.1.1',
+  [SAFE_FEATURES.ETH_SIGN]: '>=1.1.0'
 }
 
-const isEnabledByVersion = (feature: FEATURES, version: string): boolean => {
-  if (!(feature in FEATURES_BY_VERSION)) {
-    return true
+// Note: gatewau returns `SafeInfo['version']` as `null` for unsupported contracts
+export const hasSafeFeature = (feature: SAFE_FEATURES, version: string | null): boolean => {
```

Comment:
I would make `version` a required parameter. If it is `undefined` or `null` it should be handled outside of the function.
```suggestion
export const hasSafeFeature = (feature: SAFE_FEATURES, version: string): boolean => {
```